pkg/util/child.go

```diff
@@ -0,0 +1,30 @@
+// +build !linux
+
+/*
+Derived from:
+
+https://github.com/kubernetes/kubernetes
+
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"os"
+)
+
+// childSignals are the allowed signals that can be sent to children in Windows to terminate
```

Comment:
We release binaries to `linux`, `darwin` and `windows`. This file applies to both `darwin` and `windows`, therefore the comment is off-by-one.

I think `darwin` could be closer to the `linux` side, though. And well, `windows` too.

We could have a single `child.go` file that will build for all platforms we support, since `darwin` and `windows` support all of the signals we're listing right now in `child_linux.go`.

```console
$ GOOS=windows godoc syscall | egrep 'SIG(TERM|INT|HUP|QUIT)'
    SIGHUP  = Signal(0x1)
    SIGINT  = Signal(0x2)
    SIGQUIT = Signal(0x3)
    SIGTERM = Signal(0xf)
$ GOOS=darwin godoc syscall | egrep 'SIG(TERM|INT|HUP|QUIT)'
    SIGHUP    = Signal(0x1)
    SIGINT    = Signal(0x2)
    SIGQUIT   = Signal(0x3)
    SIGTERM   = Signal(0xf)
```